components/Embed.tsx

```diff
@@ -1,23 +1,23 @@
-import React from "react";
+import React from "react"
 
 interface Props {
-  src: string;
-  title?: string;
-  width?: number;
-  height?: number;
+  src: string
+  title?: string
+  width?: number
+  height?: number
 }
 
 export function Embed({ src, title, width, height }: Props) {
   return (
     <iframe
-      referrerPolicy="no-referrer"
-      sandbox="allow-forms allow-scripts"
+      sandbox='allow-forms allow-scripts allow-same-origin'
+      allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
```

Comment:
What does this mean?

Reply:
YouTube Embed was broken with `referrerPolicy` and I was also thinking about blocking of those things and if we are extremely strict, websites might break. So I've added this and later we'll figure out what is necessary and what is not. :)